ui/src/components/providers.tsx

```diff
@@ -39,7 +39,17 @@ export const Providers = ({ children }: { children: ReactNode }) => {
   return (
     <AuthProvider {...oidcConfig}>
       <QueryClientProvider client={queryClient}>{children}</QueryClientProvider>
-      <Toaster />
+      <Toaster
```

Comment:
I believe currently the UI is broken after the first commit, as `Toaster` has been migrated to the toasts are still executed with `useToast`, right?

I would squash all three commits to one.